docs/eventing/broker/rabbitmq-broker/README.md

```diff
@@ -78,7 +78,25 @@ will be `true` in the output of the following command:
 For more information about configuring the `RabbitmqCluster` CRD, see the
 [RabbitMQ website](https://www.rabbitmq.com/kubernetes/operator/using-operator.html).
 
-## Create a RabbitMQ broker config object
+## Reference an external RabbitMQ Cluster
```

Comment:
```suggestion
## Connect a RabbitMQ cluster
```
Suggestion on title naming - I think the current title makes it a bit confusing since all rabbitmq clusters technically would be external, and "connecting" gives a better idea of what the user is actually doing here I think?

Reply:
True, I'll rephrase it a bit better